lms/static/sass/discussion/_mixins.scss

```diff
@@ -114,21 +113,20 @@
 
 // extends - content - text overflow by ellipsis
 %cont-truncated {
-  @include box-sizing(border-box);
+  box-sizing: border-box;
   overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
 }
 
 @mixin forum-post-label($color) {
   @extend %t-weight4;
-  @include font-size(9);
+  font-size: 9px;
```

Comment:
We should use one of our defined size variables, or use rem/em. Also, 9px is tiny.